### PR TITLE
Rocket and missile effect update

### PR DESCRIPTION
**Pull request by Ollieollieolllie**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp [hunk 1]
```diff
@@ -195,179 +47,24 @@ class CfgCloudlets
...
-        simulation = "particles";
-        type = "Missile2";
-        position[] = {0,0,0};
-        intensity = 1;
-        interval = 1;
-        lifeTime = 1;
-    };
-};
-
-class fza_ah64_deploysmoke
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp:339]: Didn't we just implement deploysmoke - isn't it used for flechettes?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp (hunk 1 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp:339]: oops

Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp:339]: Corrected

Comment (mattysmith22) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp:339]: You re-added at a different point to where they are so they still show up in the diff - needs moving back above CfgLights

Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgCloudlets.hpp:339]: corrected

**Review by BradMick — APPROVED** [review]
Good to Go

**Review by mattysmith22 — APPROVED** [review]

**Pull request merged**